app/controllers/RiderController.java

```diff
@@ -19,14 +27,101 @@ public RiderController(RiderRepository riderRepository) {
         this.riderRepository = riderRepository;
     }
 
-    public Result index() {
-        return ok("RIDER REPO");
+    public CompletionStage<Result> getRiders() {
+        return riderRepository.getAllRiders().thenApplyAsync(riders -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)